src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -12,81 +23,101 @@
 @WebServlet("/user")
 public class UserServlet extends HttpServlet {
 
-  // Dummy list of people @TODO deprecate this
-  private List<PersonObject> people;
+  private DatastoreService datastore;
 
   @Override
-  // Instantiate the dummy list of people
+  // Instantiate datastore
   public void init() {
-    people = new ArrayList<PersonObject>();
-    ArrayList<String> projectIDs = new ArrayList<String>();
-    projectIDs.add("OpenSesame");
-
-    ArrayList<String> ObiSkills = new ArrayList<String>();
-    ObiSkills.add("Meme god");
-    ObiSkills.add("HTML wrangler");
-    PersonObject Obi =
-        new PersonBuilder()
-            .name("Obi")
-            .gitHubID("Obinnabii")
-            .description("Obi is awesome.")
-            .interestTags(ObiSkills)
-            .projectIDs(projectIDs)
-            .buildPerson();
-    people.add(Obi);
-
-    ArrayList<String> SamiSkills = new ArrayList<String>();
-    SamiSkills.add("Stone carver");
-    SamiSkills.add("Bootstrap convert");
-    PersonObject Sami =
-        new PersonBuilder()
-            .name("Sami")
-            .gitHubID("Sami-2000")
-            .description("Sami is fun.")
-            .interestTags(SamiSkills)
-            .projectIDs(projectIDs)
-            .buildPerson();
-    people.add(Sami);
-
-    ArrayList<String> RichiSkills = new ArrayList<String>();
-    RichiSkills.add("Minecraft boss");
-    RichiSkills.add("React wizard");
-    PersonObject Richi =
-        new PersonBuilder()
-            .name("Richi")
-            .gitHubID("Richie78321")
-            .description("Richi is cool.")
-            .interestTags(RichiSkills)
-            .projectIDs(projectIDs)
-            .buildPerson();
-    people.add(Richi);
+    datastore = DatastoreServiceFactory.getDatastoreService();
   }
 
   @Override
-  // get a specific user. return null if not found. TODO: User Validation
+  // Get a specific user. return null if not found. TODO: User Validation
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String userString = request.getParameter("user");
-
-    PersonObject result =
-        people.stream()
-            .filter(person -> userString.equals(person.getGitHubID()))
-            .findAny()
-            .orElse(null);
-    if (result == null) {
-      sendRawTextError(
-          response, HttpServletResponse.SC_BAD_REQUEST, "invalid user or user does not exist");
+    String userGithub = request.getParameter("githubID");
+    Key userKey = KeyFactory.createKey(PersonObject.ENTITY_NAME, userGithub);
+
+    PersonObject userObject;
+
+    try {
+      userObject = toPersonObject(datastore.get(userKey));
+    } catch (Exception e) {
+      response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Person does not exist in Datastore");
+      userObject = null;
```

Comment:
Can we just just `return` here? We already sent an error response so not sure why we need to send a null object.
```suggestion
      return;
```